devolving_music/views/get_song_pair.py

```diff
@@ -23,8 +23,9 @@ def get(self, _request, event: Event):
         scores_list = scores.get_scores()
         info_list = SongScores.get_info_sort(list(scores_list.values()))
```

Comment:
As stated in another comment, this sort should be moved inside `weighted_lowest_info`.